### PR TITLE
make TA form live

### PR DESCRIPTION
**Pull request by smwatts**
When reviewing this PR:

1. make sure that you have pip installed datetime (again, will add this to requirements along with testing instructions once this PR is merged)
2. Run python manage.py makemigrations
3. Run python manage.py migrate
4. Run python manage.py collectstatic

This PR is for the "make TA form live" functionality:

screen shot 2018-02-05 at 5 45 13 pm
screen shot 2018-02-05 at 5 45 32 pm

Screenshots show the form when it is open & closed for applicants.

Also changed some uppercase function names to lowercase at @j2kan's request

### CODE REVIEW

File: taworks/taform/templates/taform/application.html [hunk 1]
```diff
@@ -13,12 +13,39 @@
...
+        <tr>
+            <td><b>Last status change:</b></td>
+            <td>{{status_date}}&emsp;</td>
+            <td><form action="{% url 'application' %}" method="POST">
+                {% csrf_token %}
+                <input type="submit" name="app_status" value="Change Application Status"/>
+            </form></td>
+        </tr>
+    </table>
 {% endif %}
```
Comment (amyleblond) [inline taworks/taform/templates/taform/application.html:35]: I might be missing something , but is this end if doing anything?

**Review by amyleblond — DISMISSED** [review]
works well sarah. Had one comment about an 'endif' line that I don't think is being used -- but nothing to block!

When I ran I noticed that the initial intro page will still open -- should we talk about making the 'closed' message appear there or just on the actual application? Let me know what you think!

File: taworks/taform/templates/taform/application.html (hunk 1 above)
Comment (smwatts) [reply taworks/taform/templates/taform/application.html:35]: The way GitHub is displaying it makes it hard to see, but I believe its the endif for your "If AC" statement

screen shot 2018-02-05 at 7 08 08 pm

**Review by j2kan — DISMISSED** [review]
lgtm

**Comment by j2kan** [comment]
+1 on adding to intro page too.

**Review by j2kan — APPROVED** [review]

**Comment by smwatts** [comment]
I think having the intro page available all the time makes sense because it has good info about the TA process. I think we should ask Smucker to give more direction so people know when applications will open and close based on the intro form.

**Pull request merged**